packages/ecc-client-ga4gh-tes/src/components/tes-runs/defintion/tesRuns.template.ts

```diff
@@ -33,11 +33,11 @@ const template = html<TESRuns>`
         :value=${(x) => x.stateInput}
         @input=${(x, c) => x.handleStateInput(c.event)}
       >
-        <fast-option value="ALL">All</fast-option>
-        <fast-option value="COMPLETE">Completed</fast-option>
-        <fast-option value="PROCESSING">Processing</fast-option>
-        <fast-option value="SYSTEM_ERROR">Error</fast-option>
-        <fast-option value="CANCELED">Cancelled</fast-option>
+        <fast-option value="ALL">ALL</fast-option>
+        ${repeat(
+          () => stateOption,
+          html`<fast-option value=${(x) => x}>${(x) => x}</fast-option>`
```

Comment:
please take another look at this. Also make sure to test your components after making a change before you push, cause this would have been easily caught.

Reply:
What are you talking about exactly :sweat_smile: , if the "ALL" option, then that's intentional.
If done please resolve.

Reply:
@SalihuDickson